test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -78,6 +80,34 @@ public class ContentResultTest
             Assert.Equal("text/plain; charset=utf-7", httpContext.Response.ContentType);
         }
 
+        [Fact]
+        public async Task ContentResult_DisablesResponseBuffering_IfBufferingFeatureAvailable()
```

Comment:
I did not add it explicitly as buffering is not *on* by default and all the remaining tests run without it.